src/app/Navbar.tsx

```diff
@@ -1,18 +1,20 @@
 import Polyglot from 'node-polyglot'
-import { Accessor, JSXElement, Setter, Show, Signal } from 'solid-js'
+import { Accessor, JSXElement, mergeProps, Setter, Show, Signal } from 'solid-js'
 
 import { AbiItem } from 'web3-utils'
 import * as contractkit from '@celo/contractkit'
 import Web3 from '@celo/contractkit/node_modules/web3'
-import { Contract } from '@celo/contractkit/node_modules/web3-eth-contract'
 import WalletConnectProvider from '@walletconnect/web3-provider'
 
 import { getI18N } from './i18n'
 import * as constants from './constants'
 import managerABI from '../../artifacts/contracts/Manager.sol/Manager.json'
+import { Manager } from '../types/contracts/Manager'
+import { ExampleProjectTemplate } from '../types/contracts/projects'
 
 export let kit: contractkit.ContractKit
-export let managerContract: Contract
+export let managerContract: Manager
```

Comment:
this should be improved as I'd like to have a single object that contains all contracts instances, will be done in the future